site/app/templates/forum/CreatePost.twig

```diff
@@ -5,7 +5,7 @@
             {% if userGroup <= 2 and activeThreadAnnouncement %}
 
                 {% set expiringVisual = (expiring) ? 'active-thread-remove-expiring-announcement' : 'active-thread-remove-announcement' %}
-                <a class="{{expiringVisual}} key_to_click" tabindex="0" onClick="alterAnnouncement({{ activeThread.id }}, 'Are you sure you want to unpin this thread?', 0, '{{ csrf_token }}')" title="Unpin Thread" aria-label="Unpin Thread"><i class="fas fa-thumbtack reverse_golden_hover"></i></a>
+                <a class="{{expiringVisual}} key_to_click" tabindex="0" onClick="alterAnnouncement({{ activeThread.id }}, 'Are you sure you want to unpin this thread?', 0, '{{ csrf_token }}')" title="Pinned thread -- will expire soon" aria-label="Pinned thread -- will expire soon"><i class="fas fa-thumbtack"></i></a>
```

Comment:
Per the [W3C spec](https://www.w3.org/TR/WCAG20-TECHS/H33.html) for accessibility for `a` tags titles:

> The title attribute is used to provide additional information to help clarify or further describe the purpose of a link.

Let's revert this change.

Reply:
Should I just revert the aria-label or the title as well?

Reply:
Both aria-label and title should be used, where the former is going to be what's read aloud by a screen reader and the latter used when mousing over. Ideally, both should describe what the link is going to _do_ with possible additional information, as you did in follow-up commits :+1:

Thanks!